docs/advanced/inflation.md

```diff
@@ -2,7 +2,7 @@
 
 ---
 
-New decred is created each time a PoW miner discovers a new block - this is commonly referred to as the "block reward". The block reward is split three ways:
+New blocks are discovered by the PoW miners roughly every 5 minutes, and each time this occurs some new decred is created.  This "block reward" is split three ways:
```

Comment:
I don't love how 'some new decred is created' sounds.  Maybe 'new decred are created'?